### ISSUE TITLE
Unable to run GPU interactive example on RADIS Lab and local environment

### ISSUE THREAD

**Issue by TranHuuNhatHuy** [radis/radis#515]
### 🐛 Describe the bug

Currently the script introduced in gallery example of [Real-time GPU Accelerated Spectra (Interactive)](https://radis.readthedocs.io/en/latest/auto_examples/plot_gpu_widgets.html) does not run on either RADIS Lab or on my local environment.

The script:
```python
from radis import SpectrumFactory
from radis.test.utils import getTestFile
from radis.tools.database import load_spec
from radis.tools.plot_tools import ParamRange

# This spectrum is significantly absorbed by atmospheric CO2
# so it will never match the synthetic spectrum.
# TODO: find different spectrum for this example.

my_file = getTestFile("CO2_measured_spectrum_4-5um.spec")  # for the example here
s_exp = load_spec(my_file)

s_exp.crop(4120, 4790).plot(Iunit="mW/cm2/sr/nm")


sf = SpectrumFactory(
    2150,
    2450,  # cm-1
    molecule="CO2",
    isotope="1,2,3",
    wstep=0.002,
)

sf.fetch_databank("hitemp")

s = sf.eq_spectrum_gpu_interactive(
    var="radiance",
    Tgas=ParamRange(300.0, 2500.0, 1100.0),  # K
    pressure=ParamRange(0.1, 2, 1),  # bar
    mole_fraction=ParamRange(0, 1, 0.8),
    path_length=ParamRange(0, 1, 0.2),  # cm
    slit_FWHM=ParamRange(0, 1.5, 0.24),  # cm-1
    emulate=True,  # if True, runs CPU code on GPU. Set to False or remove to run on the GPU
    plotkwargs={"nfig": "same", "wunit": "nm"},
)
print(s)
```

When I set `emulate = True` which used CPU, the last traceback in error log was:
```
File /srv/conda/envs/notebook/lib/python3.8/site-packages/radis/lbl/gpu.py:469, in gpu_init(v_arr, dxG, dxL, iso, v0, da, gamma, na, S0, El, Mm_arr, Q_intp_list, verbose, gpu)
    466     from numpy import complex64, float32, zeros
    468     array = lambda arr: arr
--> 469     from radis_cython_extensions import set_init_params
    471 init_h.v_min = np.min(v_arr)  # 2000.0
    472 init_h.v_max = np.max(v_arr)  # 2400.0

ModuleNotFoundError: No module named 'radis_cython_extensions'
```

When I set `emulate = False` which used GPU, the last traceback become:
```
File /srv/conda/envs/notebook/lib/python3.8/site-packages/radis/lbl/gpu.py:449, in gpu_init(v_arr, dxG, dxL, iso, v0, da, gamma, na, S0, El, Mm_arr, Q_intp_list, verbose, gpu)
    446 # -----------------------------------------------------
    448 if gpu:
--> 449     from cupy import RawModule, array, complex64, float32, zeros
    451     cuda_fname = join(getProjectRoot(), "lbl", "gpu.cpp")
    452     with open(cuda_fname, "rb") as f:

ModuleNotFoundError: No module named 'cupy'
```

It is noted that the same errors occurred on my local environment (MacOS Monterey 12.5.1) too. I tried installing `cupy` on my device but was unsuccessful while building wheel for cupy (setup.py).

### 🎲 Radis version

0.13.0

### 💻 Operating system

MacOS

**Comment by dcmvdbekerom** [comment]
Short answer, it's not a bug but per say but an issue with the RADIS installation on the user side;

The example is really meant as a GPU demo, and in order to run it as such you need three things:
- An NVIDIA GPU (no support for other GPU's as of now, and not likely in the near future)
- CUDA toolkit: https://developer.nvidia.com/cuda-downloads
- Cupy package (Python's way of interacting with the CUDA toolkit): https://docs.cupy.dev/en/stable/install.html
I recommend to install in that order, and install a version specific cupy, i.e. not ~`pip install cupy`~ but `pip install cupy-cuda11x`, assuming you downloaded the latest CUDA toolkit (v11.7).

The `emulate_GPU` mode is a compatibility mode to run GPU code when any of the three above isn't present, most importantly for running non-regression tests at a server, but also for spotting bugs during development. 

Having said that, there's nothing stopping anyone from using it for producing spectra, but the way it works is that the C++ code for the GPU (Specifically https://github.com/radis/radis/blob/develop/radis/lbl/gpu.cpp]) is compiled for the GPU. This compilation is attempted during installation RADIS, along with compilation of some other code that speeds up RADIS in regular CPU mode as well. The compiled code is packaged together in a package called `radis_cython_extensions`.

For some reason, in your case the C code didn't get built during RADIS setup and `radis_cython_extensions` wasn't built. It's possible that in this case the build isn't triggered, in which case we need to pack the binaries with the installation files.

How did you install RADIS? (I'm assuming `pip install radis`?)

**Comment by TranHuuNhatHuy** [comment]
Thank you @dcmvdbekerom for your reply.

I installed RADIS both by using `pip install radis`, and by `pip install -e .` with the developer version as introduced in [RADIS Developer Guide](https://radis.readthedocs.io/en/latest/dev/developer.html#install). I ran the above GPU script outside the developer repo (in "Downloads" folder, to be precise), perhaps this was the reason.

I just attempted to run it inside the developer repo with `emulate = True`, and it worked. However, there was no interactive window that showed up as expected in the example, just the terminal logs running.
